apps/greenhouse-management/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "greenhouse-management",
-  "version": "1.0.7",
+  "version": "1.0.8",
```

Comment:
We have meanwhile already version 1.0.8. You should rebase from main and you will see it.
https://assets.juno.qa-de-1.cloud.sap/?__s=(assets~Foverview:(assetName:greenhouse~Fmanagement,assetVersion:1.0.8,navItem:%27%27,panelOpened:++,panelTabIndex:+0,tabIndex:+1))